packages/buendia-dashboard/data/usr/share/buendia/dashboard/backups

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+# Copyright 2015 The Project Buendia Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License.  You may obtain a copy
+# of the License at: http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distrib-
+# uted under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, either express or implied.  See the License for
+# specific language governing permissions and limitations under the License.
+
+set -e
+
+usb_devices=$(ls /dev/sd* | wc -l)
+if [ $usb_devices -eq 2 ]; then
+	usb_status="USB device connected ($(ls /dev/sd* | sed -e 's/\/dev\/[a-z][a-z][a-z]$//') )"
+else
+	usb_status="No USB device recognised. If you have one plugged in, try disconnecting the cable and reconnecting it."
+fi
+
+buendia_status=$(cat /home/buendia/server_status.txt)
+
+if [ $buendia_status -eq 2 ]; then
```

Comment:
You can use bash `case` statements for this: http://tldp.org/LDP/Bash-Beginners-Guide/html/sect_07_03.html.

Also, as is, you should be quoting the `$buendia_status` variable in these tests because you're using `[` as opposed to `[[`. See http://serverfault.com/a/52050/145418